en/docs/install-and-setup/setup/mi-setup/updating-mi.md

```diff
@@ -0,0 +1,46 @@
+# Production Checklist
+
+Once you download and install WSO2 Streaming Integrator, you may need to update its default configurations based on your requirements.
```

Comment:
The file name is "updating-mi" but the content seems to be about streaming integrator.

Reply:
Added Update Product content.